Libraries/LibGUI/TableView.cpp

```diff
@@ -177,11 +177,12 @@ void TableView::keydown_event(KeyEvent& event)
     if (event.is_accepted())
         return;
 
-    if (is_editable() && edit_triggers() & EditTrigger::AnyKeyPressed && !event.text().is_empty()) {
+    auto is_delete = event.key() == Key_Delete || event.key() == Key_Backspace;
+    if (is_editable() && edit_triggers() & EditTrigger::AnyKeyPressed && (event.code_point() != 0 || is_delete)) {
```

Comment:
Someone explain to me why `event.code_point() == '.'` when `event.key() == Key_Delete`.

Reply:
I don't know. That's strange. The code point is derived from the current character map fwiw.